backend/habitat/viewsets.py

```diff
@@ -1328,17 +1339,90 @@ def region_report_data(request):
+                'amp-network',
+                'amp-park',
+                'data-report-minimap',
+                'data-report-boundary-network-simplified',
+                'data-report-boundary-simplified',
+            ]
+        ).order_by('sort_key'),
+        many=True
+    ).data
+
+    layers = LayerSerializer(
+        Layer.objects.filter(
+            id__in=[v['layer'] for v in keyed_layers]
+        ).prefetch_related(
+            'category',
+            'data_classification',
+            'organisation',
+            'server_type'
+            ),
+        many=True
+    ).data
+
+    data["all_layers"] = list(reversed([next(layer for layer in layers if layer["id"] == keyed_layer['layer']) for keyed_layer in keyed_layers if keyed_layer['keyword'] == 'data-report-minimap-panel1']))
+    data["all_layers_boundary"] = [layer for layer in layers if layer['id'] in [keyed_layer['layer'] for keyed_layer in keyed_layers if keyed_layer['keyword'] == 'data-report-minimap-panel1-boundary']][0]
+    data["public_layers"] = list(reversed([next(layer for layer in layers if layer["id"] == keyed_layer['layer']) for keyed_layer in keyed_layers if keyed_layer['keyword'] == 'data-report-minimap-panel2']))
+    data["public_layers_boundary"] = [layer for layer in layers if layer['id'] in [keyed_layer['layer'] for keyed_layer in keyed_layers if keyed_layer['keyword'] == 'data-report-minimap-panel2-boundary']][0]
+    data["app_boundary_layer"] = [layer for layer in layers if layer['id'] in [keyed_layer['layer'] for keyed_layer in keyed_layers if keyed_layer['keyword'] == ('amp-park' if park != None else 'amp-network')]][0]
+    data["pressures"] = PressureSerializer(Pressure.objects.filter(region_report=rr.id).prefetch_related('layer'), many=True).data
+
+    data["minimap_layers"] = [{'label': keyed_layer['description'], 'layer': next(layer for layer in layers if layer['id'] == keyed_layer['layer'])} for keyed_layer in keyed_layers if keyed_layer['keyword'] == 'data-report-minimap']
+
+    # Get the boundary geometry
+    boundary_simplified = [layer for layer in layers if layer['id'] in [keyed_layer['layer'] for keyed_layer in keyed_layers if keyed_layer['keyword'] == ('data-report-boundary-simplified' if park != None else 'data-report-boundary-network-simplified')]][0]
+    params = {
+        'request':      'GetFeature',
+        'service':      'WFS',
+        'version':      '2.0.0',
+        'typeNames':    boundary_simplified['layer_name'],
+        'outputFormat': 'application/json',
+        'cql_filter': (f"RESNAME='{park}'" if park else f"NETNAME='{network}'")
+    }
 
-    with connections['transects'].cursor() as cursor:
-        cursor.execute(SQL_GET_PARK_SQUIDLE_URL if park else SQL_GET_NETWORK_SQUIDLE_URL, [park or network])
-        entry = cursor.fetchone()
-        data["squidle_url"] = entry[0] if entry else None # some park
+    try:
+        r = http_session().get(url=boundary_simplified['server_url'], params=params)
```

Comment:
This is the sort of thing that jumps out when you're reviewing; making an http call from another view (we're not doing microservices, so we don't have that kind of infrastructure).  Apart from the general inlining, I wonder if this wouldn't be better extracted into a smaller method to get the boundaries, and then that could potentially be cached?  (was this what we were talking about django's caching infrastructure for?  I can't remember)